main.cpp

```diff
@@ -99,14 +100,127 @@ inf
 
 good to go!
 
+*/
 
+/*
+ MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
 
+ Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
+ 
+ If you didn't already: 
+    Make a pull request after you make your first commit
+    pin the pull request link and this repl.it link to our DM thread in a single message.
 
+ send me a DM to review your pull request when the project is ready for review.
 
-*/
+ Wait for my code review.
+ */
 
 #include <iostream>
 
+struct FloatType
+{
+    float* value;
+    FloatType(float v) : value(new float(v)) {}
+
+    FloatType& add(float a)
+    {
+        *value += a;
+        return *this;
+    }
+
+    FloatType subtract(float a)
+    {
+        *value -= a;
+        return *this;
+    }
+
+    FloatType multiply(float a)
+    {
+        *value *= a;
+        return *this;
+    }
+
+    FloatType divide(float a)
+    {
+        if (a == .0f  || *value == .0f)
```

Comment:
It doesn't matter if value is 0.  it matters if 'a' is 0.
- [ ] revise

Reply:
Solved